pages/home/changelog.mdx

```diff
@@ -7,6 +7,31 @@ description: "What's new at Arcade.dev"
 
 *Here's what's new at Arcade.dev!*
 
+
+## For the week ending on 2025-08-15
+
+<Note>
+  This week we enforced a new requirement for all OAuth providers: they must have a unique callback URL.  This is a minor security change, but does require you to update your OAuth configuration.  This can be updated from the dashboard.
+</Note>
+
+**Toolkits**
+* `[feature - 🚀]` Sharepoint Toolkit added ([PR #30](https://github.com/ArcadeAI/toolkits/pull/30))
+* `[feature - 🚀]` Google Slides Toolkit added ([PR #28](https://github.com/ArcadeAI/toolkits/pull/28))
+* `[feature - 🚀]` Commenting on Google Docs added ([PR #35](https://github.com/ArcadeAI/toolkits/pull/35))
+* `[bugfix - 🐛]` MS Teams: MXE improvements; fix reply chat message quotation ([PR #25](https://github.com/ArcadeAI/toolkits/pull/25))
```

Comment:
Perhaps rephrase as: "Improvements in Microsoft Teams message search tool for better Agentic experience. Fix bug when no messages match the search query."